docs/source/distributed.rst

```diff
@@ -10,10 +10,14 @@ Distributed communication package - torch.distributed
 Backends
 --------
 
-``torch.distributed`` supports three backends, each with
-different capabilities. The table below shows which functions are available
+``torch.distributed`` supports three built-in backends, each with different
+capabilities. It also supports third-party c10d backend extensions by dynamic
```

Comment:
"third-party backends via a dynamic loading mechanism."

Reply:
Let's move the mention of third party backends to a separate section below the introduction and table.